server/src/main/clients/web-browser/src/index.scss

```diff
@@ -17,37 +26,171 @@ a {
     text-decoration: none;
 }
 
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+#site-title {
+    font-family: var(--heading-font-family);
+}
+
 h1 {
-    text-align: center;
+    font-size: 48px;
+}
+
+h2 {
+    font-size: 36px;
```

Comment:
You're right, for now I was just copying sizes from directly from the wireframe but using `em` instead of `px` will make responsiveness easier in the future because we can adjust font sizes and everything else will scale with it. I'll fix that.

Reply:
Done